src/server/qgsfcgiserverrequest.cpp

```diff
@@ -106,11 +107,19 @@ QgsFcgiServerRequest::QgsFcgiServerRequest()
   setUrl( url );
   setMethod( method );
 
-  // Get accept header for content-type negotiation
-  const char *accept = getenv( "HTTP_ACCEPT" );
-  if ( accept )
+  // Fill the headers dictionary
+  const QMetaEnum headerMetaEnum = QMetaEnum::fromType<QgsServerRequest::RequestHeader>();
+  for ( int idx = 0; idx < headerMetaEnum.keyCount(); ++idx )
```

Comment:
you can use `qgsEnumMap` 
https://qgis.org/api/qgis_8h.html#a496c75b1881ae4913d017446a4858272

Reply:
+1 Thanks :-)